platform/fio/domain.go

```diff
@@ -1,10 +1,35 @@
 package fio
 
 import (
+	"strings"
+
 	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/address"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 )
 
+var domains = map[string]interface{}{
```

Comment:
can re-use `StringSet` if we had, also can add helper method to init a set with string array

Reply:
I changed to StringSet, much cleaner.
Found package "github.com/minio/minio-go/pkg/set".
There is also "golang.org/x/text/internal/stringset", but internal is not allowed.